css/details.css

```diff
@@ -1,10 +1,22 @@
 table {
-	display: inline-block;
+	display: table;
```

Comment:
inline-block reduces the width used by the table. Removing `display` would be even better indeed ^